src/api-reference/10.webhooks.md

```diff
@@ -12,10 +12,11 @@ existing one. We always send the webhook to a URL with lowercase characters.
 Therefore, it is necessary to use lowercase characters in the URL of the webhook
 call.
 
-We currently have two available hooks: `shipment_status_change` and `shipment_label_created`:
+We currently have three available hooks: `shipment_status_change`, `shipment_label_created` and `order_status_change`:
 
-* shipment_status_change: Whenever the [status] of a shipment changes this webhook will send you an update with the new value. We will not send an update with a status 1.
-* shipment_label_created: When a label is created asynchronously (for example when <ApiLink to="6_B_2">creating shipments</ApiLink> using accept header `application/vnd.shipment_label+json;charset=utf-8`) this webhook will send you a message with the URL of the label.
+* shipment_status_change: Whenever the [shipment_status] of a shipment changes, this webhook will send you an update with the new value. We will not send an update with a status 1.
+* shipment_label_created: When a label is created asynchronously (for example when <ApiLink to="6_B_2">creating shipments</ApiLink> using accept header `application/vnd.shipment_label+json;charset=utf-8`), this webhook will send you a message with the URL of the label.
+* order_status_change: Whenever the [order_status] of an order changes, this webhook will send you an update with the new value. We will not send an update with a status 1.
```

Comment:
"We will not send an update with a status 1." kan verwijderd worden

Reply:
Aangepast